zellij-server/Cargo.toml

```diff
@@ -14,15 +14,17 @@ ansi_term = "0.12.1"
 async-trait = "0.1.50"
 base64 = "0.13.0"
 byteorder = "1.4.3"
+bytes = "1.6.0"
 daemonize = "0.5"
 serde_json = "1.0"
 unicode-width = "0.1.8"
 url = "2.2.2"
-wasmer = "3.1.1"
-wasmer-wasi = "3.1.1"
+wasmtime = "20.0.2"
+wasmtime-wasi = "20.0.2"
 cassowary = "0.3.0"
 zellij-utils = { path = "../zellij-utils/", version = "0.41.0" }
 log = "0.4.17"
+tokio = "1.37.0"
```

Comment:
Why tokio? We're already using async_std internally, I'm not thrilled about adding another async runtime...

Reply:
wasmtime-wasi has a dependency on tokio. In any case this direct dependency here can be removed I think. I used it in an earlier version of the stdio pipe implementation, but didn't remove it when I changed the implementation.